feedhenry/fh-reportingclient
# ⬆ replace bunyan with bunyan-lite, and also update fh-mbaas-client to

the latest version

To verify:

1. run `export V=1` and run `npm install` with node10 in master branch. You will see bunyan will try to build a native module called `dtrace_provider` and it should fail to build the module.
2. repeat the step with this PR and you should not the `dtrace_provider` module at all and the build should pass.
